packages/project-scripts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hpe/project-scripts",
-  "version": "1.1.2",
+  "version": "2.0.0",
```

Comment:
it wasn't clear if this would be a breaking change, but I assumed that it would be, and thus proposed a 2.0.0 version number